src/questions/chapter5.yaml

```diff
@@ -0,0 +1,1239 @@
+      order: 3
+
+- uuid: 91f22dcc-dd72-45e5-a8bb-cf56f66a2563
+  type: MultipleChoice
+  content: Which one of the following are essential features of an object-oriented programming language? 
+    * (i) Abstraction and encapsulation 
+    * (ii) Strictly-typedness 
+    * (iii) Type-safe property coupled with sub-type rule 
+    * (iv) Polymorphism in the presence of inheritance
+  hint: noHint
+  explanation: Abstraction, Encapsulation, Polymorphism and Inheritance are the essential features of a OOP Language.
+  answerKeys:
+    - B
+  topic: OOP
+  subTopics:
+    - Abstraction
+    - Encapsulation
+    - Polymorphism
+    - Inheritance
+  difficultyLevel: Low
+  choices:
+    - content: (i), (ii) only
+      key: A
+      order: 0
+    - content: (i) and (iv) only
+      key: B
+      order: 1
+    - content: (i), (ii) and (iv) only
+      key: C
+      order: 2
+    - content: (i), (iii) and (iv) only
+      key: D
+      order: 3
+
+- uuid: 18ee66e0-f94c-48a8-b23b-1ccad0de3431
+  type: MultipleChoice
+  content: Which of the following statements regarding the features of the object-oriented approach to databases are true? 
+    * (a)The ability to develop more realistic models of the real world. 
+    * (b)The ability to represent the world in a non-geometric way. 
+    * (c)The ability to develop databases using natural language approaches. 
+    * (d)The need to split objects into their component parts. 
+    * (e)The ability to develop database models based on location rather than state and behaviour.
+  hint: noHint
+  explanation: Features of the object-oriented approach to databases 
+    * The ability to develop more realistic models of the real world. 
+    * The ability to represent the world in a non-geometric way. 
+    * The ability to develop databases using natural language approaches. 
+    So, option (A) is correct.
+  answerKeys:
+    - A
+  topic: OOP
+  subTopics:
+  difficultyLevel: Medium
+  choices:
+    - content: (a), (b) and (c)
+      key: A
+      order: 0
+    - content: (b), (c) and (d)
+      key: B
+      order: 1
+    - content: (a), (d) and (e)
+      key: C
+      order: 2
+    - content: (c), (d) and (e)
+      key: D
+      order: 3
+
+- uuid: 46a72ca3-f513-417c-bf3e-099c9610f7e9
+  type: MultipleChoice
+  content: Which objects can only be accessed internally from the current contract instances?
+  hint: noHint
+  explanation: Private objects can only be accessed internally from the current contract instances.
+  answerKeys:
+    - B
+  topic: Encapsulation
+  subTopics:
+  difficultyLevel: Low
+  choices:
+    - content: public
+      key: A
+      order: 0
+    - content: private
+      key: B
+      order: 1
+    - content: internal
+      key: C
+      order: 2
+    - content: external
+      key: D
+      order: 3
+
+- uuid: 5d6b3cca-0774-4bea-90bc-c0fb6bb0c0c0
+  type: MultipleChoice
+  content: Which of the following does not belong to state visibility?
+  hint: hint
+  explanation:
```

Comment:
Add explanation why `External` doesn't belong to state visibility. Also please change the question type to `SingleChoice`